docs/examples/03_building_custom_image.ipynb

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Ideally we should also add this to the e2e tests too

Reply:
Created separate issue for that https://github.com/Qiskit-Extensions/qiskit-serverless/issues/1316

Not sure yet how to make e2e, since it will not work in local mode and in docker compose as well.